assets/js/hooks/Mapper/components/mapInterface/widgets/SystemSignatures/constants.ts

```
@@ -40,45 +44,56 @@ export const GROUPS: Record<SignatureGroup, GroupType> = {
-  // RUSSIAN
-  [SignatureKindRU.CosmicSignature]: SignatureKind.CosmicSignature,
-  [SignatureKindRU.CosmicAnomaly]: SignatureKind.CosmicAnomaly,
-  [SignatureKindRU.Structure]: SignatureKind.Structure,
-  [SignatureKindRU.Ship]: SignatureKind.Ship,
-  [SignatureKindRU.Deployable]: SignatureKind.Deployable,
-  [SignatureKindRU.Drone]: SignatureKind.Drone,
-};
+// Flatten the structure for backward compatibility
+export const MAPPING_GROUP_TO_ENG: Record<string, SignatureGroup> = Object.entries(LANGUAGE_GROUP_MAPPINGS).reduce(
+  (result, [, mappings]) => {
+    return { ...result, ...mappings };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)